lib/elixir/lib/string.ex

```diff
@@ -1478,6 +1481,9 @@ defmodule String do
       iex> String.at("elixir", -10)
       nil
 
+      iex> String.at("\u0065\u0301lixir", 0)
```

Comment:
Sure, but I was trying to illustrate the multicodepoint situation: Combining Acute Accent (U+0301) is a second codepoint, rather than a two-byte single codepoint Latin Small Letter E with Acute Accent (U+00e9).